docs/en/latest/deployments/azure.md

```diff
@@ -38,6 +38,8 @@ This document explains how to install Ingress APISIX on [Azure AKS](https://docs
 ```shell
 cd /path/to/apisix-helm-chart
 helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo add apisix https://charts.apiseven.com
+# Use `hem search repo apisix` to search charts about apisix
 helm dependency update ./charts/apisix
 helm install apisix ./charts/apisix \
```

Comment:
here should also be changed.

Reply:
Also, the installation of apisix-ingress-controller should also change to use the online repo.

Reply:
Update the commit.